dialdbg/src/parse.rs

```diff
@@ -0,0 +1,326 @@
+use super::strings;
+use anyhow::{anyhow, Result};
+use chrono::{DateTime, Duration, FixedOffset};
+use std::{fmt, fs, io, net::SocketAddr, path::PathBuf};
+
+const DEVELOPMENT: Option<&'static str> = option_env!("DIALDBG_DEVELOPMENT");
+
+#[derive(Debug, Default)]
+struct GRPCResult {
+    // The mDNS address queried (None if mDNS was not used in connection establishment).
+    mdns_address: Option<SocketAddr>,
+    // The time taken to query mDNS (None if mDNS was not used in connection establishment or
+    // query failed).
+    mdns: Option<Duration>,
```

Comment:
(minor) `mdns` is a bit non-descriptive as a name. Can we update to be more clear? Same applies to the `WebRTCResult` type.

Reply:
Updated to `mdns_query`, but I don't know if that's any more descriptive. I could suffix most of these fields with `time` or `duration` if that sounds better to you?